dispatch/static/manager/src/js/pages/ItemIndexPage.js

```diff
@@ -24,6 +25,10 @@ export default class ListItemsPageComponent extends React.Component {
       query.listItem = this.props.location.query.listItem
     }
 
+    if (typeof this.props.extraQueryHandler === 'function') {
```

Comment:
Instead of checking the type here, it would be better to add these props to `propTypes` and then just do `if (this.props.extraQueryHandler)`

Alternatively, you could use `defaultProps` and a stub function like so to avoid the conditional entirely:

```js

ListItemsPageComponent.defaultProps = {
  extraQueryHandler: (query) => query
}
```